### PR TITLE
(feat) KPH3-7035 : Updating the waiting in line  UI component for deceased persons

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

waitinginline

## Related Issue
see:- https://thepalladiumgroup.atlassian.net/browse/KHP3-7035

## Other

*None.*

### CODE REVIEW

**Comment by donaldkibet** [comment]
Thanks @its-kios09 we should have the admit and release as an overflow menu items to follow carbon design languge

File: packages/esm-morgue-app/src/utils/utils.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
   return new URL(makeUrl(path), window.location.toString());
 }
+
+/**
+ * Formats a date to "October 20 2024, 01:53 PM" format.
+ * @param {string|Date|number} date - The date to format, can be a Date object, timestamp, or ISO string.
+ * @returns {string} - The formatted date string.
+ */
+export function formatToReadableDate(date) {
+  return dayjs(date).format('MMMM DD YYYY, hh:mm A');
```
Comment (donaldkibet) [inline packages/esm-morgue-app/src/utils/utils.ts:14]: Recommend against this, for UI uniformity. We have helper functions from `@openmrs/esm-framework` if we need this we should extend the date formats in `esm-framework`

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @its-kios09, left some suggestions.

**Comment by its-kios09** [comment]
> Thanks @its-kios09 we should have the admit and release as an overflow menu items to follow carbon design languge

Let me revert to` formatDate() `function

File: packages/esm-morgue-app/src/tables/generic-table.component.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const GenericTable: React.FC<GenericTableProps> = ({ rows, headers, actionColumn
                         {header.header}
                       </TableHeader>
                     ))}
-                    {actionColumn && <TableHeader>Action</TableHeader>}
+                    {<TableHeader>{''}</TableHeader>}
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-morgue-app/src/tables/generic-table.component.tsx]: sorted

File: packages/esm-morgue-app/src/tables/waiting-queue.component.tsx [hunk 3]
```diff
@@ -6,22 +6,21 @@ import { Tag, Button, DataTableSkeleton, OverflowMenu, OverflowMenuItem } from '
 import styles from './generic-table.scss';
 import { useTranslation } from 'react-i18next';
 import { useDeceasedPatient, useVisitType } from '../hook/useMorgue.resource';
+import { formatToReadableDate } from '../utils/utils';
 
 export const WaitingQueue: React.FC = () => {
   const { data: deceasedPatients, error, isLoading } = useDeceasedPatient();
   const { t } = useTranslation();
-  const fromHospital = t('fromHosp', 'From Hospital');
+  const fromHospital = t('waitingInLine', 'Waiting In Line');
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-morgue-app/src/tables/waiting-queue.component.tsx:14]: sorted

File: packages/esm-morgue-app/src/tables/waiting-queue.component.tsx [hunk 4]
```diff
@@ -6,22 +6,21 @@ import { Tag, Button, DataTableSkeleton, OverflowMenu, OverflowMenuItem } from '
...
 
   const genericTableHeader = [
-    { header: 'S/No', key: 'serialNumber' },
-    { header: 'Deceased Name', key: 'name' },
-    { header: 'ID No ', key: 'Ids' },
+    { header: 'Patient Name', key: 'name' },
     { header: 'Gender', key: 'gender' },
-    { header: 'Dob', key: 'birthdate' },
+    { header: 'Identifier ', key: 'Ids' },
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-morgue-app/src/tables/waiting-queue.component.tsx:19]: sorted

File: packages/esm-morgue-app/src/utils/utils.ts (hunk 1 above)
Comment (its-kios09) [reply packages/esm-morgue-app/src/utils/utils.ts:14]: reverted

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**